modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/home/components/ProjectsList/components/ProjectCard/components/StatusDescription/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import classNames from 'classnames';
+import i18n from '../../../../../../../../common/I18n';
+import {SLA_STATUS_TYPES} from '../../../../../../../../common/utils/constants';
+import getDateCustomFormat from '../../../../../../../../common/utils/getDateCustomFormat';
+
+const StatusDescription = ({compressed, slaCurrentEndDate, status}) => {
+	const statusReport = {
+		[SLA_STATUS_TYPES.active]: i18n.translate('ends-on'),
+		[SLA_STATUS_TYPES.future]: i18n.translate('starts-on'),
+		[SLA_STATUS_TYPES.expired]: i18n.translate('ended-on'),
+	};
```

Comment:
Hi @wandersonvicente, should be good this variables comes before the component. Because it's a static variable. Don't need to be re-rendered.